cdv/cmds/sim_utils.py

```diff
@@ -0,0 +1,525 @@
+    docker_mode: bool,
+) -> None:
+    # either return passed through fingerprint or get one
+    fingerprint = select_fingerprint(fingerprint, mnemonic_string, docker_mode)
+    if fingerprint is None:
+        # user cancelled wizard
+        return
+    # create chia directory & get config.
+    print("Creating chia directory & config...")
+    config = create_chia_directory(root_path, fingerprint, farming_address, plot_directory, auto_farm, docker_mode)
+    # Pre-generate plots by running block_tools init functions.
+    print("Please Wait, Generating plots...")
+    print("This may take up to a minute if you are on a slow machine")
+
+    await generate_plots(config, root_path, fingerprint)
+    # final messages
+    farming_address = config["simulator"]["farming_address"]
+    print(f"\nFarming & Prefarm reward address: {farming_address}\n")
+    print("Configuration Wizard Complete.")
+    print("Starting Simulator now...\n\n")
+    await start_async(root_path, ("simulator",), False)
+    # now we create make sure the simulator has a genesis block
+    print("Please wait, generating genesis block.")
+    while True:
+        try:
+            num_blocks = await execute_with_simulator(None, root_path, get_current_height, False)
+        except ClientConnectorError:
+            await asyncio.sleep(0.25)
+        else:
+            if num_blocks == 0:
+                await execute_with_simulator(None, root_path, farm_blocks, False, 1, True, farming_address)
+                print("Genesis block generated, exiting.")
+            else:
+                print("Genesis block already exists, exiting.")
+            break
+
+
+def print_coin_record(
+    name: str,
+    address_prefix: str,
+    mojo_per_unit: int,
+    coin_record: CoinRecord,
+) -> None:
+    from datetime import datetime
+
+    chia_amount = Decimal(int(coin_record.coin.amount)) / mojo_per_unit
+    coin_address = encode_puzzle_hash(coin_record.coin.puzzle_hash, address_prefix)
+    print(f"Coin 0x{coin_record.name.hex()}")
+    print(f"Wallet Address: {coin_address}")
+    print(f"Confirmed at block: {coin_record.confirmed_block_index}")
+    print(f"Spent: {f'at Block {coin_record.spent_block_index}' if coin_record.spent else 'No'}")
+    print(f"Coin Amount: {chia_amount} {name}")
+    print(f"Parent Coin ID: 0x{coin_record.coin.parent_coin_info.hex()}")
+    print(f"Created at: {datetime.fromtimestamp(float(coin_record.timestamp)).strftime('%Y-%m-%d %H:%M:%S')}\n")
+
+
+async def print_coin_records(
+    config: Dict[str, Any],
+    node_client: SimulatorFullNodeRpcClient,
+    include_reward_coins: bool,
+    include_spent: bool = False,
+) -> None:
+    import sys
+
+    from chia.cmds.units import units
+
+    coin_records: List[CoinRecord] = await node_client.get_all_coins(include_spent)
+    coin_records = [coin_record for coin_record in coin_records if not coin_record.coinbase or include_reward_coins]
+    address_prefix = "txch"  # we are never on mainnet

```

Comment:
Even if we're never on mainnet, we shouldn't prevent people from using whatever prefix they want.